Junk4Dollars-appUITests/Junk4Dollars_appUITests.swift

```diff
@@ -0,0 +1,32 @@
+import XCTest
+
+class Junk4Dollars_appUITests: XCTestCase {
+    var app: XCUIApplication!
+
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
```

Comment:
What does this do?

Reply:
This will stop a single UI test from running if it encounters a failure, but other tests will still run. I think this is mainly important because it's simulating user clicks, so if something fails early on, the remainder of that test will likely fail anyway and be a waste of time.